### DISCUSSION TITLE
Projects (Beta) - Allow filtering by `assignee: current-user`

### DISCUSSION THREAD

**Discussion by defunctzombie** [community/community#5732] [Projects and Issues] [answered]
The new views feature in Projects Beta is great! We are able to easily create views around specific workflows (i.e. no:label or no:assignee for triage workflows). We'd like the ability to create a view that shows the current user's assigned issues. This would give our team members quick access within the board to see their assigned issues and work in progress.

**Comment by JoJossd** [comment]
+1

**Comment by ganny-gitaa** [comment]
I was searching through discussions for an answer to the exact same question. 
+1

**Comment by espendk** [comment, marked as answer]
I think this has been solved with the `@me` feature: https://github.com/github/feedback/discussions/5769

**Reply by FezVrasta** [reply]
I wonder what @me thinks about this feature, he can't send links pointing to its own account anymore 😅
